msa-persistence/src/main/java/cz/muni/fi/pa165/monsterslayeragency/entities/Hero.java

```diff
@@ -2,26 +2,43 @@
 
 import cz.muni.fi.pa165.monsterslayeragency.enums.Skill;
 
+import javax.persistence.*;
 import java.util.Arrays;
 import java.util.Objects;
 
 /**
  * @author Ludovit Kopcsanyi
  */
+@Entity
 public class Hero {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private long id;
 
+	@Column(nullable = false)
+	@ManyToOne(targetEntity = User.class, fetch = FetchType.LAZY)
 	private User user;
-	private String name;
+
+	@Column(nullable = false, unique = true)
+	private String heroName;
```

Comment:
I agree with @krytoss. I would definitely prefer this attribute to be called `name` instead of `heroName`.

Reply:
I don't think there is an explicit problem with `heroName`, because this suggests this is more of a sort of a nickname, than the actual name of the hero. I think it doesn't matter if the attribute is called `name` or `heroName`.